.github/pull_request_template.md

```diff
@@ -1,59 +1,101 @@
--->
+**User Story or Bug Summary:**
+<!-- Please copy-paste the brief user story or bug description that this PR is intended to address. -->
 
-### Change Details
 
-<!-- Add detailed discussion of changes here: -->
-<!-- This is likely a summary, or the complete contents, of your commit messages -->
+### What Does This PR Do?
 
-### Acceptance Validation
+<!--
+Add detailed description & discussion of changes here.
+The contents of this section should be used as your commit message (unless you merge the PR via a merge commit, of course).
 
-<!-- What should reviewers look for to determine completeness -->
+Please follow standard Git commit message guidelines:
+* First line should be a capitalized, short (50 chars or less) summary.
+* The rest of the message should be in standard Markdown format, wrapped to 72 characters.
+* Describe your changes in imperative mood, e.g. "make xyzzy do frotz" instead of "[This patch] makes xyzzy do frotz" or "[I] changed xyzzy to do frotz", as if you are giving orders to the codebase to change its behavior.
+* List all relevant Jira issue keys, one per line at the end of the message, per: <https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html>.
 
-<!-- Insert screenshots if applicable (drag images here) -->
+Reference: <https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project>.
+-->
 
-### Feedback Requested
 
-<!-- What type of feedback you want from your reviewers? -->
+### What Should Reviewers Watch For?
 
-### External References
+<!--
+Add some items to the following list, or remove the entire section if it doesn't apply for some reason.
+
+Common items include:
+* Is this likely to address the goals expressed in the user story?
+* Are any additional documentation updates needed?
+* Are there any unhandled and/or untested edge cases you can think of?
+* Is user input properly sanitized & handled?
+* Does this make any backwards-incompatible changes that might break end user clients?
+* Can you find any bugs if you run the code locally and test it manually?
+-->
 
-<!-- For example: replace xxx with the JIRA ticket number: -->
+If you're reviewing this PR, please check these things, in particular:
 
-- [BLUEBUTTON-xxx](https://jira.cms.gov/browse/BLUEBUTTON-xxx)
+* TODO
 
-### Security Implications
 
-<!-- Does the change deal with PII/PHI at all? What should reviewers look for in
-terms of security concerns? -->
+### What Security Implications Does This PR Have?
 
-- [ ] new software dependencies
+Submitters should complete the following questionnaire:
 
-<!-- If yes, list the new dependencies and briefly note any relevant security impacts -->
+* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
+    * Does this PR add any new software dependencies? **Yes** or **No**.
```

Comment:
It sounds a bit verbose to emphasize that even dependencies introduced for development and testing use, e.g. debugging libs / plugins, python packages / java jars used solely for unit tests, they are also subject to this screening process.

In past months, e.g. BB2 introduced ptvsd (dev dependency for remote debugging), httmock / beautifulsoap4 for unit testing etc.

Reply:
Yeah it could be made to exclude certain types of dependencies like those only needed for dev/testing.  Feedback for that should go here since I just copied this: https://github.com/CMSgov/cms-oeda-dasg/pull/1